walk-in-interview/src/main/java/com/google/job/data/JobPayment.java

```diff
@@ -0,0 +1,99 @@
+package com.google.job.data;
+
+/** Class that represents the payment details of a job. */
+public final class JobPayment {
+    private final int min;
+    private final int max;
+    private final PaymentFrequency paymentFrequency;
+
+    private int hashCode;
```

Comment:
recommend `volatile` see comment below in getHashCode()

Reply:
Fixed